controller/webapp/management/commands/backpopulate_from_saved_job_request.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from controller.lib import database
+from controller.models import Job
+from controller.queries import get_saved_job_request
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Development utility for creating and submitting a RAP without having a
```

Comment:
I start every management command by copying an existing one too!

I guess it doesn't really matter given this is a one-shot command.